### PR TITLE
Silence write-file and write auxiliary file

### PR DESCRIPTION
**Pull request by pietbrauer**
Silences:

```bash
Write auxiliary files
write-file /Users/jenkins/Jenkins/workspace/iphone-app-pullrequests-unit-tests/Build/Intermediates/Pods.build/Debug-iphonesimulator/Pods-JLRoutes.build/Pods-JLRoutes.hmap
```

### CODE REVIEW

**Comment by pietbrauer** [comment]
I am pretty sure that the tests are broken on the `log-formatting-blacklist` branch

**Comment by supermarin** [comment]
@pietbrauer thanks!

would you mind opening a PR against `master`? I'm cherry-picking commits slowly from `log-formatting-blacklist` into master, and should probably wrap up the work on that asap

File: lib/xcpretty/parser.rb [hunk 1]
```diff
@@ -181,6 +181,13 @@ module Matchers
     # $2 file_name
     TOUCH_MATCHER = /^Touch\s(.*\/([\w+\.]+))/
 
+    # @regex Captured groups
+    # $1 file_path
+    WRITE_FILE_MATCHER = /^write-file\s(.*)/
+
+    # @regex Captured groups
+    WRITE_AUXILIARY_FILE = /^Write auxiliary files/
```
Comment (supermarin) [inline lib/xcpretty/parser.rb]: nitpick: s/WRITE_AUXILIARY_FILE/WRITE_AUXILIARY_FILES

It's a plural, singular made me think there should be a filename parameter

File: lib/xcpretty/parser.rb [hunk 2]
```diff
@@ -339,6 +346,10 @@ def parse(text)
         formatter.format_tiffutil($1)
       when TOUCH_MATCHER
         formatter.format_touch($1, $2)
+      when WRITE_FILE_MATCHER
+        formatter.format_write_file($1)
+      when WRITE_AUXILIARY_FILE
+        formatter.format_write_auxiliary_file
```
Comment (supermarin) [inline lib/xcpretty/parser.rb]: nit: same as above

**Comment by pietbrauer** [comment]
Closing this one in favour of #128

**Pull request closed without merging**